applications/FluidDynamicsApplication/custom_processes/distance_modification_process.cpp

```diff
@@ -87,14 +87,15 @@ void DistanceModificationProcess::CheckDefaultsAndProcessSettings(Parameters &rP
 {
     Parameters default_parameters( R"(
     {
-        "model_part_name"                        : "",
-        "distance_factor"                        : 2.0,
-        "distance_threshold"                     : 0.001,
-        "continuous_distance"                    : true,
-        "check_at_each_time_step"                : true,
-        "avoid_almost_empty_elements"            : true,
-        "deactivate_full_negative_elements"      : true,
-        "recover_original_distance_at_each_step" : false
+        "model_part_name"                             : "",
+        "distance_factor"                             : 2.0,
```

Comment:
This `distance_factor` is here because of legacy reasons (we used to use it in our very first implementation). I search for it and there is only two concurrencies (one in this process and one in its Python apply one).

Maybe it's a good idea to take the chance to remove it in this PR.

Reply:
I see!

I also wanted to switch the NodalH calculation from historical to non historical (was thinking to do so in another PR), would you do it here as well? 

@jcotela, is what you did in #4850 enough to define the nodalH calculation to NonHistorical safely?

Reply:
Yes, but I did not change existing code, so you still need to call the new function wherever appropriate. Also, if you find other missing MPI operations, please let me know and I'll look into adding them.

Reply:
> I see!
> 
> I also wanted to switch the NodalH calculation from historical to non historical (was thinking to do so in another PR), would you do it here as well?
> 
> @jcotela, is what you did in #4850 enough to define the nodalH calculation to NonHistorical safely?

This I'd do it in another PR since it is a potential source of errors.